README.md

```diff
@@ -222,6 +221,74 @@ Use the following options if you need to configure a private link with Coralogix
 
+### Metadata
+
+The metadata features decribed below are only available in `coralogix-aws-shipper v1.1.0` and later.
+
+The `AddMetadata` parameter allows you to add metadata to the log message. The metadata is added to the log message as a JSON object. The metadata is specific to the integration type. For example, for S3, the metadata is `s3.object.key` and `s3.bucket`. For CloudWatch, the metadata is `cw.log.group` and `cw.log.stream`. See table below for full list of metadata.
+
+| Integration Type | Metadata Key             | Description                           |
+|------------------|--------------------------|---------------------------------------|
+| S3               | s3.bucket                | The name of the S3 bucket             |
+| S3               | s3.object.key            | The key/path of the S3 object         |
+| CloudWatch       | cw.log.group             | The name of the CloudWatch log group  |
+| CloudWatch       | cw.log.stream            | The name of the CloudWatch log stream |
+| Cloudwatch       | cw.owner                 | The owner of the log group            |
+| Kafka            | kafka.topic              | The name of the Kafka topic           |
+| MSK              | kafka.topic              | The name of the Kafka topic           |
+| Kinesis          | kinesis.event.id         | The kinesis event ID                  |
+| Kinesis          | kinesis.event.name       | The kinesis event name                |
+| kinesis          | kinesis.event.source     | The kinesis event source              |
+| kinesis          | kinesis.event.source_arn | The kinesis event source ARN          |
+| Sqs              | sqs.event.source         | The sqs event source/queue            |
+| Sqs              | sqs.event.id             | The sqs event id                      |
+| Ecr              | ecr.scan.id              | The ecr scan id                       |
+| Ecr              | ecr.scan.source          | The ecr scan source                   |
+
+Note that metadata is not added by default. You must specify the metadata keys you want in the `AddMetadata` parameter.
+
+For example, if you want to add the bucket name and key name to the log message, you would set the `AddMetadata` parameter to `s3.object.key,s3.bucket`.
+
+Some metadata keys will overlap as some integrations share the same metadata. For example, both Kafka and MSK have the same metadata key `kafka.topic` or both Kinesis and Cloudwatch metadata will be added in cases where a Cloudwatch log stream is being ingested from a Kinesis stream.
+
+##### Dynamic Subsystem or Application Name
+
+As of `v1.1.0`,you can use dynamic values for the Application and Subsystem Name parameters based on the internal metadata defined above.
+
+To do accomplish this, you can use the following syntax:
+
+```
+ApplicationName/SubsystemName: {{ metadata.key | r'regex' }}
```

Comment:
IMO this example with a `/` is confusing. It would be cleared to have separate examples for `ApplicationName` and `SubsystemName`